### ISSUE TITLE
Usage of product.json To Use VSCode's Marketplace Doesn't Work

### ISSUE THREAD

**Issue by SkyWolfXP** [VSCodium/vscodium#2056]
**Describe the bug**
I have followed the guide on where and how to create the `product.json` file, But wasn't able to install extensions from VSCode's Marketplate within VSCodium.

**To Reproduce**
1. creating a custom product.json at the following location: `%APPDATA%\VSCodium`
2. pasting:
```
{
  "extensionsGallery": {
    "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
    "itemUrl": "https://marketplace.visualstudio.com/items",
    "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
    "controlUrl": ""
  }
}
```

**Expected behavior**
Bring VSCode Marketplace extensions to VSCodium's Extensions tab

**Screenshots**
![{D5E9AE90-78E6-4DBB-BAC1-93DDDEE57C3A}](https://github.com/user-attachments/assets/3534ce4b-58e3-4183-9678-f6c7a51f4d20)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Architecture x64
 - VSCodium Version 1.94.2.24284

**Comment by daiyam** [comment]
Yep, it's broken in `1.94`... I will fix it soon.

**Comment by daiyam** [comment]
It has been fixed in #2057 and released in [`1.94.2.24286`](https://github.com/VSCodium/vscodium/releases/tag/1.94.2.24286).
